react/TextField/TextField.js

```diff
@@ -88,11 +88,12 @@ export default class TextField extends Component {
   }
 
   renderInput() {
-    const { inputProps = {}, id } = this.props;
+    const { inputProps = {}, id, valid } = this.props;
     const { ref } = inputProps;
     const allInputProps = {
       ...combineClassNames(inputProps, styles.input),
       ...(id ? { id } : {}),
+      ...{ 'aria-invalid': valid === false },
```

Comment:
Good point and not really think of any case but was just thinking around what happened if somebody did have it already set and updated..... But I guess it would still all work just double set so not a huge deal?